### ISSUE TITLE
Final validation and training loss?

### ISSUE THREAD

**Issue by wmcnally** [mks0601/TF-SimpleHumanPose#64]
Do you have on hand the final validation/training losses? I'm still struggling to reproduce the 70.4 AP with TF2. Highest I've gotten so far is 68.7... My validation loss is around 30. Thanks.

**Comment by mks0601** [comment]
Sorry I don't have it :( I think the network architecture is quite simple, so data pre-processing such as affine transformation can matter.

**Comment by wmcnally** [comment]
Ok. No problem! Thanks for the quick reply.
